Ignore insignificant file system changes by default (#5953)
ocaml/dune: CHANGES.md

- Fix handling of the `(deps)` field in `(test)` stanzas when there is an
  `.expected` file. (#5952, #5951, fixes #5950, @emillon)

- Ignore insignificant filesystem events. This stops RPC in watch mode from

>   flashing errors when file system events such as changes in the `.git/`
>   directory occur. (#5953, @rgrinberg)

Reviewer: ```suggestion
  flashing errors on insignificant file system events such as changes in the
  `.git/` directory. (#5953, @rgrinberg)
```
